hackclub.com.yaml

```diff
@@ -726,6 +726,11 @@ boreal:
   type: CNAME
   value: cname.vercel-dns.com.
 
+busan:
+  ttl: 600
+  type: CNAME
+  value: counterspell-busan.netlify.app.
+
```

Comment:
Deven told me that we aren't giving hackclub.com subdomains to Counterspell events. You can always PR into the https://github.com/hackclub/counterspell repo
```suggestion
```

Reply:
Yup - thanks!